src/utils/postAttestationProfile.js

```
@@ -0,0 +1,88 @@
+            reject(err);
+        }
+
+        const network = require('ocore/network.js');
+        const composer = require('ocore/composer.js');
+
+        const sanitizedProfile = {
+            provider: String(provider),
+            ...profile
+        };
+
+        let objMessage = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'objMessage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)